platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygTextView.swift

```diff
@@ -44,6 +44,9 @@ protocol WysiwygTextViewDelegate: AnyObject {
 public protocol MentionDisplayHelper { }
 
 public class WysiwygTextView: UITextView {
+    private var lastInputMode: String?
+    private(set) var isDictationRunning = false
```

Comment:
Maybe it's better if this is property computed based on the `lastInputMode`?

Reply:
it needs to be changed also according to two override methods for ending the recording and when the recording fails

Reply:
Maybe those should clean up the `lastInputMode` instead?